ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/TopicStreamTest.java

```diff
@@ -79,7 +80,7 @@ public void shouldMatchAvroFormatter() throws Exception {
     final GenericData.Record avroRecord = new GenericData.Record(schema);
     avroRecord.put("str1", "My first string");
 
-    expect(schemaRegistryClient.register(anyString(), anyObject())).andReturn(1);
+    expect(schemaRegistryClient.register(anyString(), any(Schema.class))).andReturn(1);
```

Comment:
this uses easymock, not mockito - so it's causing the tests to fail

Reply:
Yes, I fixed it. 

It also relies on certain SR calls within the `KafkaAvroSerializer`.  Those calls will no longer be the same in the future, so I'll temporarily disable this test right before I check in my SR changes.